### DISCUSSION TITLE
Communication > Means of Communication > In-Game Movement/Actions

### DISCUSSION THREAD

**Discussion by giscus** [pgpais/live-framework-for-cooperative-games#99] [Framework Discussions]
# Communication > Means of Communication > In-Game Movement/Actions

https://www.lfcooperativegames.com/reports/new

**Comment by hg65we** [comment]
Indirectly, you can see on the map shots ocurring between an ally and nothing (you can't see the enemy your ally sees) signaling an enemy's position
